cibyl/cli/validator.py

```diff
@@ -83,7 +84,7 @@ def _consistent_system(self, system):
 
         return True
 
-    def _system_has_valid_sources(self, system):
+    def _system_has_valid_sources(self, system: object) -> bool:
```

Comment:
This is what I mean when talking about inconsistency, the docstring and the type hint are hinting for two different types here. The docstring one is more precise as is the one I would go with. So, it is a matter of having one or the other, but not both.

Reply:
It feels like we are re-discussing the whole idea of type hinting while we already did it